operations/model.go

```diff
@@ -20,11 +20,12 @@ import (
 const localConfigPath = ".evergreen.local.yml"
 
 type ClientProjectConf struct {
```

Comment:
This should probably be a global setting, not per project.

Reply:
done.